lib/publify/api.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+
+module Publify
+  class API
+    def self.latest_links(limit)
+      uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+      response = Net::HTTP.get uri
```

Comment:
Ay, that's flagged above. This comment was about timeouts.

So, after looking into it, I discovered that by default `Net::HTTP.get` will wait forever to get a connection, and then 60 seconds for a response. Seems like we should set a shorter timeout to avoid issues with Publify cascading into a total loss of service.